server/handle_map_zxy.go

```diff
@@ -113,10 +117,17 @@ func (req HandleMapZXY) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	tile := slippy.NewTile(req.z, req.x, req.y, TileBuffer, tegola.WebMercator)
 
-	//	filter down the layers we need for this zoom
+	// filter down the layers we need for this zoom
 	m = m.FilterLayersByZoom(req.z)
```

Comment:
This might be something for another pull request, but there should also be bounds checking for the map xy as well as z at this part of the request fulfillment

Reply:
Yeah, that makes sense. We should factor out [this code](https://github.com/go-spatial/tegola/blob/v0.7.0/cache/cache.go#L60) into a helper class to validate zxy values.

Reply:
Well that is implemented in the parseURI method, but I meant to check if the request is within the bounds of map as specified in the server's `config.toml`

Reply:
That's a good point.